doc/introduction/tutorials.rst

```diff
@@ -119,14 +153,23 @@ Algorithms
     :description: :doc:`/gallery/gate_synthesis/GateSynthesis`
     :figure: /gallery/gate_synthesis/GateSynthesis.gif
 
-.. customgalleryitem::
-    :tooltip: Scattershot boson sampling
-    :description: :doc:`/gallery/scattershot-boson-sampling/scattershot-bs`
-    :figure: /gallery/scattershot-boson-sampling/scattershot-bs.gif
+.. raw:: html
+
+        <div style='clear:both'></div>
+        <br>
+
+Hardware
```

Comment:
Would potentially recommend putting this part above or directly below applications